### PR TITLE
Project Keyboard Alice PCB - Indicator LEDs and keymap update 

### PR DESCRIPTION
**Pull request by sidcarter**
## Description

* Update to allow Indicator LEDs to be mapped by user
* Switch to use new method to update LEDs
* Update map to use LEDs
* Disable console on keymap to allow it to work in Linux

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/projectkb/alice/alice.c [hunk 1]
```diff
@@ -1,27 +1,18 @@
 #include "alice.h"
 
+__attribute__ ((weak))
```
Comment (fauxpark) [inline keyboards/projectkb/alice/alice.c]:
```suggestion
```

File: keyboards/projectkb/alice/alice.c [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
 void matrix_init_board(void){
```
Comment (fauxpark) [inline keyboards/projectkb/alice/alice.c]:
```suggestion
void matrix_init_kb(void) {
    matrix_init_user();
    led_init_ports();
}

void led_init_ports(void) {
```

`matrix_init_board()` doesn't seem to be an actual function in QMK, and is not called anywhere. This is the usual structure of the LED init code.

Comment (sidcarter) [reply keyboards/projectkb/alice/alice.c]: I imagine cause the new suggestion doesn't compile - there's already a matrix_init_kb defined. Error when compiling

```
 |
 | .build/obj_projectkb_alice_madhatter/keyboards/projectkb/alice/alice.o: In function `matrix_init_kb':
 | /Users/sidcarter/qmk_firmware/keyboards/projectkb/alice/alice.c:3: multiple definition of `matrix_init_kb'
 | .build/obj_projectkb_alice_madhatter/keyboard.o:/Users/sidcarter/qmk_firmware/keyboards/cannonkeys/stm32f072/keyboard.c:86: first defined here
 | collect2: error: ld returned 1 exit status
 |
make[1]: *** [.build/projectkb_alice_madhatter.elf] Error 1
```

Comment (fauxpark) [reply keyboards/projectkb/alice/alice.c]: Ah... the board seems to be pulling in some stuff from the cannonkeys folder, looks like for a custom underglow implementation. So this'll do for now.

Per [this comment](https://github.com/qmk/qmk_firmware/pull/7173#issuecomment-550577236) from @zvecr, the new bitbang method works on F072. I believe there is also some backlight support on ARM now.

Comment (zvecr) [reply keyboards/projectkb/alice/alice.c]: f072 crashes if you use the regular backlight stuff (and crashes outside of the available driver implementations). `cannonkeys/stm32f072/keyboard.c` seems to be working round this by providing backlight keycodes when `BACKLIGHT_ENABLE = no`.

Comment (sidcarter) [reply keyboards/projectkb/alice/alice.c]: I'm guessing based on this conversation that I can resolve this? :)

**Comment by sidcarter** [comment]
@drashna @fauxpark are you waiting on any other changes for this PR?

**Comment by fauxpark** [comment]
Nope, sorry. Looks good. Thanks!

**Pull request merged**